src/test/java/Test/LoginTest.java

```diff
@@ -10,7 +10,8 @@ public void LoginTest() throws InterruptedException{
         pageFactory.getMenu().menu();
         pageFactory.getAdduser().start();
         pageFactory.getAddUserDetails().User();
```

Comment:
Test name do not match the test which we are writing in this class

Reply:
I'm confuse in it 
If it does not match  then it is not even showing error